rfcs/text/0000-application-monitoring-for-ga.md

```diff
@@ -0,0 +1,126 @@
+### DCP PR:
+
+# System monitoring for General Availability
+
+## Summary
+
+This RFC describes the first standard revision for monitoring the distributed systems of the Human Cell Atlas (HCA) Data Coordination Platform (DCP). The standard is scoped to the minimum viable set of monitoring infrastructure the DCP will need to reach General Availability (GA). We anticipate that future revisions will be needed to meet our Service Level Objectives (SLOs) as DCP matures.
+
+The term monitoring is used here as defined in the [Google Site Reliability Engineering (SRE)](https://landing.google.com/sre/sre-book/chapters/monitoring-distributed-systems/) program. This document borrows heavily from SRE.
+
+## Author(s)
+
+* [Matthew Weiden](mailto:mweiden@chanzuckerberg.com)
+
+## Shepherd
+
+* [Jodi Hirschman](mailto:jhirsch@broadinstitute.org)
+
+## Motivation
+
+Monitoring the distributed systems [enables operators](https://landing.google.com/sre/sre-book/chapters/monitoring-distributed-systems/#why-monitor-pWsBTZ) to analyze long-term trends in system performance, compare experiments against a baseline, alert on failures, and conduct retrospective analysis on systems (debugging). These abilities are critical to keeping software correct and available.
+
+### User Stories
+
+* As a DCP operator, I want to compare experiments against a baseline, so that I can compare design alternatives.
+* As a DCP operator, I want to conduct retrospective analysis on systems, so that I can debug systems.
+* As a DCP operator, I want to analyze long-term trends in system performance, so that I can estimate changes in cost and performance.
+* As a DCP operator, I want to be alerted of system failure, so that I know when something goes wrong and can keep the system available for our users.
+
+## Detailed Design
+
+In summary, each system critical to user-facing features in DCP should have the following.
+
+* Black-box availability monitoring using a health check
+* White-box monitoring of core system infrastructure
+* Alerts on black-box health check failures and white-box infrastructure anomalies
+* A dashboard that summarizes your white-box metrics
+
+The following subsections recommend concrete implementations of each requirement.
+
+### Overview of monitoring tooling
+
```

Comment:
This is a long list.  Which if these technologies fits into which category above (black, white, alerts, dashboards)?  Perhaps another column would help classify them.